examples/erc721-consecutive/src/constructor.sol

```diff
@@ -2,6 +2,10 @@
 pragma solidity ^0.8.21;
 
 contract Erc721ConsecutiveExample {
+    string private _name;
```

Comment:
Changes to solidity constructor bring an error from `koba`. Without them and without additional fields inside `Erc721`, consecutive extension's tests can pass.